### DISCUSSION TITLE
Docs: When running locally EXTENSIONS_PATH should be set to ./../extensions

### DISCUSSION THREAD

**Discussion by soulseekah** [directus/directus#18044] [Archive]
### Describe the Bug

By default `EXTENSIONS_PATH` is set to `./extensions`. However, when running locally via `pnpm --filter directus dev` as per the docs, the CWD is actually inside the ./api/ directory, where the ./extensions directory does not exist. It's one level up.

There is an extensions directory under api/ as well, however the docs https://docs.directus.io/extensions/creating-extensions.html#installing-an-extension do not mention installing it under api.

The documentation should either point this out on https://docs.directus.io/contributing/running-locally.html (set `EXTENSIONS_PATH` to `./../extensions` or https://docs.directus.io/extensions/creating-extensions.html#installing-an-extension should mention that in local development mode the extensions go into ./api/extensions not ./extensions.

### To Reproduce

Follow https://docs.directus.io/contributing/running-locally.html and https://docs.directus.io/extensions/creating-extensions.html#installing-an-extension

Expect INFO: Loaded extensions: example
Get: nothing, extensions are not loaded

### Hosting Strategy

Self-Hosted (Custom)

**Comment by rijkvanzanten** [comment]
I'm not entirely sure if this applies to every use case though 🤔 For local development, there's no guarantee that the folder structure is exactly that right?

**Reply by soulseekah** [reply]
Well, following the repository layout, the default doesn't work. The documentation should at least mention (probably in the config section) that for extensions to work on local installs (here: https://docs.directus.io/contributing/running-locally.html#_5-create-a-env-file) we need to supply the full path to extensions as the default "./extensions" is actually under "api/extensions".

Either way some hints in the docs would help greatly. Maybe in the Extensions doc section here https://docs.directus.io/extensions/creating-extensions.html#local-extension there's already a note but it's not helpful for Local installs at all.

I mean in hindsight it's quite logical that the `EXTENSIONS_PATH` setting in api/.env would actually be relative to the api directory (doh), but several man hours where our team independently tried to get extensions to load starting several times from scratch and following the docs to the letter left us all laughing nervously :)

I would include something like:

> Note: in Local Installation environments the `EXTENSION_PATH` setting is set to "./extensions/", which is relative to the location of the .env file (i.e. the api/ subdirectory), i.e. api/extensions is where extensions will be loaded from.

Or, we can rephrase this part:

> To install an extension locally, you have to move the output from the dist/ folder into your project's ./extensions/extension-folder/extension-name/ folder. 

To mention that "your project" is actually "api/" when running locally. And not the repo root.

Hope this helps others running locally out there. With at least this discussion existing :) because the only other one https://github.com/directus/directus/discussions/12160 which I'm sure was related pertains to a Docker setup, where "./extensions" is actually the root of the build and not the api/ subdirectory.

Thank you for your patience and your hard work!
